src/Uart/Uart.h

```diff
@@ -34,29 +34,30 @@
 
 #include <UartRegisters.h>
 #include <ByteFifo.h>
-#include <stdint.h>
 #include <stdbool.h>
+#include <stdint.h>
+#include <stdlib.h>
```

Comment:
Why do we need stdlib? I am not sure we will be able to use it in the qualified version of RTEMS...